package.json

```diff
@@ -33,6 +33,7 @@
   "author": "@patmood",
   "license": "ISC",
   "dependencies": {
+    "ci": "^2.3.0",
```

Comment:
I think this is breaking the github actions. Is it necessary?

Reply:
Uh oh... sorry about that, I don't remember making this change.

I think I ran `npm i ci` by accident instead of `npm ci`